sql/pipelines/src/main/scala/org/apache/spark/sql/pipelines/graph/CoreDataflowNodeProcessor.scala

```diff
@@ -198,22 +197,20 @@ private class FlowResolver(rawGraph: DataflowGraph) {
               )
           }
       }
-    result
   }
 
   private def convertResolvedToTypedFlow(
       flow: UnresolvedFlow,
       funcResult: FlowFunctionResult): ResolvedFlow = {
-    val typedFlow = flow match {
-      case f: UnresolvedFlow if f.once => new AppendOnceFlow(flow, funcResult)
-      case f: UnresolvedFlow if funcResult.dataFrame.get.isStreaming =>
+    flow match {
+      case _ if flow.once => new AppendOnceFlow(flow, funcResult)
```

Comment:
Are we confident that not checking for `UnresolvedFlow` doesn't change behavior?